### ISSUE TITLE
Encode and sort Content files by path?

### ISSUE THREAD

**Issue by sergey-tihon** [eridem/vscode-nupkg#6]
Do not you think about sorting content/files alphabetically by path?
```
Contents:
  - File:    lib/net45/FsUnit.NUnit.dll
  - File:    lib/net45/FsUnit.NUnit.XML
  - File:    lib/net45/FsUnit.NUnit.dll.mdb
  - File:    tools/install.ps1
  - File:    FsUnit.nuspec
  - File:    package/services/metadata/core-properties/coreProp.psmdcp
  - File:    _rels/.rels
  - File:    [Content_Types].xml
  - File:    lib/netstandard1.6/FsUnit.NUnit.dll
```

`+` displayed as encoded in `Newtonsoft.Json`

```
Contents:
  - File:    _rels/.rels
  - File:    Newtonsoft.Json.nuspec
  - File:    lib/net20/Newtonsoft.Json.dll
  - File:    lib/net20/Newtonsoft.Json.xml
  - File:    lib/net35/Newtonsoft.Json.dll
  - File:    lib/net35/Newtonsoft.Json.xml
  - File:    lib/net40/Newtonsoft.Json.dll
  - File:    lib/net40/Newtonsoft.Json.xml
  - File:    lib/net45/Newtonsoft.Json.dll
  - File:    lib/net45/Newtonsoft.Json.xml
  - File:    lib/netstandard1.0/Newtonsoft.Json.dll
  - File:    lib/netstandard1.0/Newtonsoft.Json.xml
  - File:    lib/portable-net40%2Bsl5%2Bwp80%2Bwin8%2Bwpa81/Newtonsoft.Json.dll
  - File:    lib/portable-net40%2Bsl5%2Bwp80%2Bwin8%2Bwpa81/Newtonsoft.Json.xml
  - File:    lib/portable-net45%2Bwp80%2Bwin8%2Bwpa81/Newtonsoft.Json.dll
  - File:    lib/portable-net45%2Bwp80%2Bwin8%2Bwpa81/Newtonsoft.Json.xml
  - File:    tools/install.ps1
  - File:    [Content_Types].xml
  - File:    package/services/metadata/core-properties/a3e3ee1d17b24ff88f746fd01ca36053.psmdcp
```

**Comment by eridem** [comment]
Fixed the **sort** on https://github.com/eridem/vscode-nupkg/releases/tag/v0.0.6
ping @sergey-tihon 

I will verify the encoding next version

**Comment by sergey-tihon** [comment]
Thank you @eridem !

**Comment by eridem** [comment]
@sergey-tihon fixed on 0.0.8 and added support for profile names too 👍
